resqpy/grid/moved_functions.py

```diff
@@ -0,0 +1,37 @@
+"""Submodule containing some general grid functions that have now been moved. This file will be deleted in a later release."""
+
+import warnings
+
+import resqpy.olio.grid_functions as gf
+import resqpy.property.property_kind as pk
+
+
+def establish_zone_property_kind(model):
+    """MOVED: Returns zone local property kind object, creating the xml and adding as part if not found in model."""
+    warnings.warn(
+        'This function has been moved to property/property_kind. Please update your code to the new '
+        'location.', DeprecationWarning)
+
+    return pk.establish_zone_property_kind(model)
+
+
+def find_cell_for_x_sect_xz(x_sect, x, z):
```

Comment:
Sorry Chris, on second thoughts I now feel this function should live in the same module as the find_cell_for_point_xy() method, ie. in points_functions.py